GTG/gtk/browser/main_window.py

```diff
@@ -759,6 +760,11 @@ def focus_quickentry(self, action, param):
 
         self.quickadd_entry.grab_focus()
 
+    def focus_sidebar(self, action, param):
+        """Callback to focus the sidebar widget."""
+        self.sidebar.props.visible = True
+        self.tagtreeview.grab_focus()
```

Comment:
Should the whole treeview grab the focus, or its currently selected row/iter?
(just asking theoretically, I have no idea :)

Reply:
Well as far as I can remember it worked while testing. Also, when you tab to an treeview you automatically can change the selection (and pressing tab again goes to the next widget rather than a row AFAIK), so I think this is fine. Also, trying to focus the row yourself seems quite some work, and a treeview would know what to do sensibly whenever it gets focus (otherwise it would be useless).

Reply:
Grabbing the treeview is the best option. It already retains and starts the currently selected row and it's a lot less hassle than keeping and updating the iter manually